app/src/main/java/org/gdg/frisbee/android/lead/LeadMessage.java

```diff
@@ -0,0 +1,48 @@
+package org.gdg.frisbee.android.lead;
+
+public class LeadMessage {
+    private final String mTitle;
+    private final Type mType;
+    private final String mDetails;
+    private String mLinkUrl;
+
+    private LeadMessage(Type type, final String title, final String details) {
+        mTitle = title;
+        mDetails = details;
+        mType = type;
+    }
+
+    private LeadMessage(final Type type, final String title, String details, final String linkUrl) {
+        this(type, title, details);
```

Comment:
Instead of this constructor using the other, I would make the other to use this constructor with a null linkUrl. What do you think?